src/js/tracks/text-track-settings.js

```diff
@@ -306,14 +306,14 @@ class TextTrackSettings extends ModalDialog {
    *         The DOM element that gets created.
    * @private
    */
-  createElSelect_(key, legendId = '') {
+  createElSelect_(key, legendId = '', type='label') {
```

Comment:
Linter is complaining about the `=` in `type='label'` not having spaces. That's a hard one for me to follow, too, coming from some Python experience.